rpcs3/Emu/RSX/Overlays/overlay_message_dialog.cpp

```diff
@@ -189,9 +238,18 @@ namespace rsx
 				btn_ok.translate(0, offset);
 				btn_cancel.translate(0, offset);
 			}
+			else
+			{
```

Comment:
Idk if it makes sense to only fade the dialogs that don't have progress bars. There is no real connection between the bars and our compile dialogs.
But i guess it makes sense since ssds are so fast